locale/en/locale.cfg

```diff
@@ -63,38 +63,44 @@ map_info_sub_caption=    ..diggy diggy choo choo..
+message_accident=Comfylatron: Offline player had an accident, and their corpse fell to the ground near the locomotive.
+message_silo=Nuclear silo destroyed. You looted __1__ atomic bombs. Comfylatron seized them for your own safety.
+message_game_won_restart=Comfylatron: WHOOSH... the chronotrain teleported us back in time... so let's do this again...
+message_fishmarket1=Comfylatron: So here we are. Fish Market. When they ordered the fish, they told us this location was safe. Guess we'll have to make it so.
+message_fishmarket2=Comfylatron: I hope you have enough nukes. That satellite gave us some space knowledge!
+message_fishmarket3=Comfylatron: Hey I found these nukes we looted before... you can take them.
+message_lava=Comfylatron: OOF this one is a bit hot. Better keep your distance! Or try some bricks to protect from lava? And have seen those biters? They BATHE in fire!
 message_choppy=Comfylatron: OwO what are those strange trees?!? They have ore fruits! WTF!
-message_game_lost1=The chronotrain was destroyed!
-message_game_lost2=Comfylatron is going to kill you for that...he has time machine after all!
-message_evolve=Comfylatron: Biters start to evolve faster! We need to charge forward or they will be stronger! (hover over timer to see evolve timer)
-message_quest1=Comfylatron: You know...I have big quest. Deliver fish to fish market. But this train is broken. Please help me fix the train computer!
-message_quest3=Comfylatron: Ah, we need to give this machine more power and better navigation chipset. Please bring me some additional things.
-message_quest5=Comfylatron: Finally found the main issue. We will need to rebuild whole processor. Exactly what I feared of. Just a few more things...
-message_quest6=Comfylatron: And this was last part of cpu brain done. Now we just need to synchronize our time correctly and we are done! Bring me satellite and rocket silo.
-message_quest7=Comfylatron: Time synchronized. Calculating time and space destination. Success. Jump once more and let me deliver the fish finally. This trip is getting long.
-message_game_won1=Comfylatron: Thank you with helping me on this delivery. It was tough one. I hope, that now, when all biters are dead, fish will be safe here forever...
-message_overstay=Comfylatron: Looks like you stayed on previous planet for so long that enemies on other planets had additional time to evolve!
-message_poison_defense=Comfylatron: Triggering poison defense. Let's kill everything!
+message_game_lost1=The Chronotrain was destroyed!
+message_game_lost2=Comfylatron is going to kill you for that... he has a time machine after all!
+message_evolve=Comfylatron: From now on, if we take too long to jump, the biters in our next timeline will evolve!
+message_quest1=Comfylatron: Comfylatron has...a big quest!! The train is broken. Can you help me fix the train computer? PS: bring fish to the fish market or die
```

Comment:
```suggestion
message_quest1=Comfylatron: You know...I have a big quest. Deliver fish to fish market. But this train is broken. Please help me fix the train computer!
```

Reply:
Agreed (tho what's the rationale behind broken english?)